src/seedsigner/models/settings_definition.py

```diff
@@ -699,6 +689,14 @@ class SettingsDefinition:
                       visibility=SettingsConstants.VISIBILITY__ADVANCED,
                       default_value=SettingsConstants.OPTION__ENABLED),
 
+        SettingsEntry(category=SettingsConstants.CATEGORY__SYSTEM,
+                      attr_name=SettingsConstants.SETTING__WORDLIST_LANGUAGE,
+                      abbreviated_name="wordlist_lang",
+                      display_name=_mft("Mnemonic language"),
```

Comment:
Display name should match the naming convention elsewhere: "BIP-39 mnemonic language" or "BIP-39 wordlist" or maybe just "BIP-39 language"(?).